### PR TITLE
Feature/postfix

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

**Review by adamgilbert — APPROVED** [review]
+1

File: .env [hunk 1]
```diff
@@ -17,19 +17,22 @@ HTTPS_PORT=443
 DOCKER_REGISTRY_URL=docker.sdelements.com:443/moodle
 
 # Moodle variables
-# WWWROOT MUST be IP or FQDN
-MOODLE_WWWROOT=https://localhost
-MOODLE_DATAROOT=/opt/moodle/moodledata
 MOODLE_ADMIN_EMAIL=admin@example.com
 MOODLE_ADMIN_PASS=Ch@ngeM3!
+MOODLE_DATAROOT=/opt/moodle/moodledata
```
Comment (antevens) [inline .env:22]: I think DATAROOT and WWWROOT are two separate words (DATA_ROOT, WWW_ROOT) and I think the code readability would improve with underscores.

File: conf/etc/php/7.2/fpm/php.ini [hunk 2]
```diff
@@ -1041,7 +1041,7 @@ smtp_port = 25
 
 ; For Unix only.  You may supply arguments as well (default: "sendmail -t -i").
 ; http://php.net/sendmail-path
-;sendmail_path =
+sendmail_path = "/usr/sbin/ssmtp -t"
```
Comment (antevens) [inline conf/etc/php/7.2/fpm/php.ini:1044]: It's not a path if it's got a -t in it, then it's a command.

File: docker-compose.yml [hunk 3]
```diff
@@ -27,6 +28,7 @@ services:
       PGSQL_USER: "${PGSQL_USER}"
       PGSQL_PASSWORD: "${PGSQL_PASSWORD}"
       PHP_VERSION: "${PHP_VERSION}"
+      SSMTP_REWRITEDOMAIN: "${SSMTP_REWRITEDOMAIN}"
```
Comment (antevens) [inline docker-compose.yml:31]: suggest SSMTP_REWRITE_DOMAIN

File: .env (hunk 1 above)
Comment (hrpatel) [reply .env:22]: These are the variables in Moodle so I kept them to match:

```php
$CFG->wwwroot   = 'http://34.235.152.190';
$CFG->dataroot  = '/opt/moodledata';
```

File: conf/etc/php/7.2/fpm/php.ini (hunk 2 above)
Comment (hrpatel) [reply conf/etc/php/7.2/fpm/php.ini:1044]: Take it up with PHP 😛

File: docker-compose.yml (hunk 3 above)
Comment (hrpatel) [reply docker-compose.yml:31]: Same as above, I named it to match the configuration parameter:

```
# Where will the mail seem to come from?
#RewriteDomain=
```

**Review by antevens — APPROVED** [review]

**Pull request merged**